client/src/pages/NewsPage/NewsPage.jsx

```diff
@@ -65,12 +71,14 @@ const NewsPage = () => {
 			<div className="cards-list-container">
 				{filteredNews.map((item) => {
```

Comment:
(don't commit this directly, i cannot suggest the full block)

(I have ordered these as they will be used in the Card)

```suggestion
				{filteredNews.map((newsArticle) => {
					const { id, title, date, category, summary } = newsArticle;
					const { url, alternativeText } = newsArticle.images[0];
					return (
						<Card
							key={id}
							id={id}
							image={url}
							alternativeText={alternativeText}
							title={title}
						  date={date}
              tag={category}
							summary={summary}
						/>
					);
				})}
```

Reply:
^do you want to do this?

Reply:
^do you want to do this destructuring?

(my comment was saying that I am not able to "suggest" the correct lines because GitHub says I am suggesting on deleted lines)

Reply:
oh sorry I got it wrong i will do it now